benchmarks/src/main/scala/monix/connect/benchmarks/redis/RedisHashesBenchmark.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020-2020 by The Monix Connect Project Developers.
+ * See the project homepage at: https://connect.monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.connect.benchmarks.redis
+
+import monix.connect.redis.RedisHash
+import monix.execution.Scheduler.Implicits.global
+import org.openjdk.jmh.annotations._
+
+import scala.concurrent.Await
+import scala.concurrent.duration.DurationInt
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@Measurement(iterations = 5)
+@Warmup(iterations = 1)
+@Fork(1)
+class RedisHashesBenchmark extends RedisBenchFixture {
+
+  var keysCycle: Iterator[String] = _
+
+  @Setup
+  def setup(): Unit = {
+    flushdb
+
+    val keys = (0 to maxKey).toList.map(_.toString)
+    keysCycle = Stream.continually(keys).flatten.iterator
+
+    (1 to maxKey).foreach { key =>
+      val value = getRandomString
```

Comment:
I would aim to get rid of any randomization in the benchmark scenarios. 
I think it can lead to higher variance in the results. Ideally, all runs should measure exactly the same path

Reply:
Yes, I got it. It would be better to use the same string for value as for key